src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
```

Comment:
Consider using named imports instead of importing everything from a module. This can help with code readability and avoid naming conflicts. 🤔